databricks/koalas/series.py

```diff
@@ -725,6 +726,100 @@ def count(self):
         """
         return self._reduce_for_stat_function(_Frame._count_expr)
 
+    def apply(self, func, args=(), **kwds):
+        """
+        Invoke function on values of Series.
+
+        Can be a Python function that only works on the series.
+
+        .. note:: unlike Pandas, it is required for `func` to specify return type hint.
+
+        Parameters
+        ----------
+        func : function
+            Python function to apply. Note that type hint for return type is required.
+        args : tuple
+            Positional arguments passed to func after the series value.
+        **kwds
+            Additional keyword arguments passed to func.
+
+        Returns
+        -------
+        Series
+
+        Examples
+        --------
+        Create a series with typical summer temperatures for each city.
+
+        >>> s = ks.Series([20, 21, 12],
+        ...               index=['London', 'New York', 'Helsinki'])
+        >>> s
+        London      20
+        New York    21
+        Helsinki    12
+        Name: 0, dtype: int64
+
+
+        Square the values by defining a function and passing it as an
+        argument to ``apply()``.
+
+        >>> def square(x) -> np.int64:
+        ...     return x ** 2
+        >>> s.apply(square)
+        London      400
+        New York    441
+        Helsinki    144
+        Name: square(0), dtype: int64
+
+
+        Define a custom function that needs additional positional
+        arguments and pass these additional arguments using the
+        ``args`` keyword by using `koalas_wraps`.
```

Comment:
what's up with koalas_wraps here? It is not part of the example.

Reply:
Ahh.. it's mixed with my another PR in my local. Let me revert..